### PR TITLE
add scripts for PSID tutorials

### PR DESCRIPTION
**Pull request by jd-lara**

### CODE REVIEW

File: script/4_PowerSimulationsDynamics_examples/01_omib.jl [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+# **Originally Contributed by**: Rodrigo Henriquez and José Daniel Lara
+
+# ## Introduction
+
+# This tutorial will introduce you to the functionality of `PowerSimulationsDynamics`
+# for running Power System Simulations.
+# Note that this tutorial is for `PowerSimulationsDynamics`. Current version already
+# have a dedicated function to find an equilibrium point using a Power Flow method
+# without relying in a guess of the initial condition.
```
Comment (claytonpbarrows) [inline script/4_PowerSimulationsDynamics_examples/01_omib.jl:11]:
```
# This tutorial will introduce you to the functionality of `PowerSimulationsDynamics`
# for running power system dynamic simulations.
```

File: script/4_PowerSimulationsDynamics_examples/01_omib.jl [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+# This tutorial presents a simulation of a two-bus system, with an infinite bus
```
Comment (claytonpbarrows) [inline script/4_PowerSimulationsDynamics_examples/01_omib.jl:13]: remove `,` after system

File: script/4_PowerSimulationsDynamics_examples/01_omib.jl [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+# The first step consists in initialize all packages that will be used to run the
+# simulation. All the necessary packages are listed:
+
+using PowerSimulationsDynamics
+using PowerSystems
+using Sundials
+const PSY = PowerSystems
+
+# `PowerSystems` is used to properly define the data structure, while `Sundials` is
```
Comment (claytonpbarrows) [inline script/4_PowerSimulationsDynamics_examples/01_omib.jl:28]:
```
# `PowerSystems` is used to properly define the data structure and establish an equilibrium point initial condition with a power flow routine, while `Sundials` is
```

File: script/4_PowerSimulationsDynamics_examples/01_omib.jl [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+# The next step is to create the simulation structure. This will create the indexing
+# of our system that will be used to formulate the differential-algebraic system of
+# equations. To do so, it is required to specify the perturbation that will occur in
+# the system. `PowerSimulationsDynamics` support two types of perturbations:
+
+# - Three Phase Fault
+# - Change in Reference Parameter
+
+# In here, he will use a Three Phase Fault, that is modeled by modifying the
+# admittance matrix of the system. To do so we create a NetworkSwitch perturbation
```
Comment (claytonpbarrows) [inline script/4_PowerSimulationsDynamics_examples/01_omib.jl:47]:
```
 # admittance matrix (Ybus) of the system. To do so we create a `NetworkSwitch` perturbation 
```

File: script/4_PowerSimulationsDynamics_examples/01_omib.jl [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+execute!(sim, #simulation structure
+                IDA(), #Sundials DAE Solver
+                dtmax=0.02); #Arguments: Maximum timestep allowed
+
+# In some cases, the dynamic time step used for the simulation may fail. In such case, the
+# keyword argument `dtmax` can be used to limit the maximum time step allowed for the simulation.
+
+## Step 5: Exploring the solution
+
+# After running the simulation, our simulation structure `sim` will have the solution. For
```
Comment (claytonpbarrows) [inline script/4_PowerSimulationsDynamics_examples/01_omib.jl:107]: replace `have` with `contain`

File: script/4_PowerSimulationsDynamics_examples/01_omib.jl [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+# that `sim.solution` can be used to explore the solution structure. In this case
+# `sim.solution.t` returns the vector of time, while `sim.solution.u` return the array of
```
Comment (claytonpbarrows) [inline script/4_PowerSimulationsDynamics_examples/01_omib.jl:109]: return = returns

File: script/4_PowerSimulationsDynamics_examples/01_omib.jl [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+# states. In addition, `PowerSimulationsDynamics` have two functions to obtain different
```
Comment (claytonpbarrows) [inline script/4_PowerSimulationsDynamics_examples/01_omib.jl:110]: have = has

**Review by claytonpbarrows — COMMENTED** [review]
some text changes

**Review by jd-lara — COMMENTED** [review]
@claytonpbarrows do we need to execute `literate_file` as part of the PR? or that is done automatically by the CI?

**Pull request merged**